runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_crafting.json

```
@@ -66,12 +78,36 @@
       "name": "Leather Boots",
       "xp": 16.3
     },
+    {
+      "level": 7,
+      "icon": 1789,
+      "name": "Unfired pie dish",
+      "xp": 10
+    },
+    {
+      "level": 7,
+      "icon": 2313,
+      "name": "Pie dish",
+      "xp": 10
+    },
     {
       "level": 7,
       "icon": 11068,
       "name": "Gold Bracelet",
       "xp": 25
     },
+    {
+      "level": 8,
+      "icon": 1791,
+      "name": "Unfired bowl",
+      "xp": 15
```

This should be 18, with the firing of the bowl being 15: https://oldschool.runescape.wiki/w/Unfired_bowl, https://oldschool.runescape.wiki/w/Bowl
```suggestion
      "xp": 18
```